pkg/controller/account/iam.go

```diff
@@ -166,72 +165,6 @@ func (r *ReconcileAccount) CreateSecret(reqLogger logr.Logger, account *awsv1alp
 	return nil
 }
 
-// BuildSTSUser sets up an IAM user with the proper access and creates secrets to hold credentials
-// Takes a logger, an awsSetupClient for the signing token, an awsClient for, an account CR to set ownership of secrets, the namespace to create the secret in, and a role to assume with the creds
-// The awsSetupClient is the client for the user in the target linked account
-// The awsClient is the client for the user in the payer level account
-func (r *ReconcileAccount) BuildSTSUser(reqLogger logr.Logger, awsSetupClient awsclient.Client, awsClient awsclient.Client, account *awsv1alpha1.Account, nameSpace string, iamRole string) (string, error) {
```

Comment:
Confirmed that this method is not called anywhere.

(Perhaps this cleanup belongs in a separate PR?)

Reply:
So this got removed accidentally, so I'll need to re-add this in.